DigitalOffice/Services/AuthenticationService/Startup.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using MassTransit;
+using FluentValidation;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.AuthenticationService.Models;
+using LT.DigitalOffice.AuthenticationService.Commands;
+using LT.DigitalOffice.AuthenticationService.Validators;
+using LT.DigitalOffice.AuthenticationService.Token;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders.Interface;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+using LT.DigitalOffice.AuthenticationService.Commands.Interfaces;
+using LT.DigitalOffice.Broker.Responses;
+
+namespace LT.DigitalOffice.AuthenticationService
+{
+    public class Startup
+    {
+        public IConfiguration Configuration { get; }
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            ConfigJwt(services);
+
+            services.AddHealthChecks();
+
+            services.AddControllers();
+
+            ConfigRabbitMq(services);
+
+            services.AddMassTransitHostedService();
+
+            ConfigBrokerSenders(services);
+            ConfigCommands(services);
+            ConfigValidators(services);
+        }
+
+        private void ConfigJwt(IServiceCollection services)
+        {
+            var signingKey = new SigningSymmetricKey();
+            var signingDecodingKey = (IJwtSigningDecodingKey)signingKey;
+
+            services.AddSingleton<IJwtSigningEncodingKey>(signingKey);
+            services.AddSingleton<NewToken>();
+
+            services.Configure<TokenOptions>(Configuration.GetSection("TokenSettings"));
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+            .AddJwtBearer(options =>
+            {
+                options.RequireHttpsMetadata = true;
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ValidateIssuer = true,
+                    ValidIssuer = Configuration.GetSection("TokenSettings:TokenIssuer").Value,
+                    ValidateAudience = true,
+                    ValidAudience = Configuration.GetSection("TokenSettings:TokenAudience").Value,
+                    ValidateLifetime = true,
+                    IssuerSigningKey = signingDecodingKey.GetKey(),
+                    ValidateIssuerSigningKey = true
+                };
+            });
+        }
+
+        private void ConfigRabbitMq(IServiceCollection services)
+        {
+            string serviceId = "123";
```

Comment:
I think it's best to move it in appsettings, because easier to change and no need to search in the code

Reply:
Fixed